### PR TITLE
Local guilty commit bug fix and optimization

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**
**Optimization:**

- [x] Enabled all support backend for guilty commit search
- [x] Remove `FREEZE` parameter which will be controlled by `backend` parameter
- [x] Use benchmarks/dynamo/runner.py TABLE for backend command line options

**Bug fix:**

- [x] Fix local guilty commit search bug: previously, local guilty commit search job only checked out END_COMMIT, but pytorch version in docker is still be START_COMMIT.

**Before fix:**
END_COMMIT will fail for first time try test expect performance: https://inteltf-jenk.sh.intel.com/job/inductor_local_guilty_commit_search/145
image

**After fix:**
END_COMMIT success for first time try test.: https://inteltf-jenk.sh.intel.com/job/inductor_local_guilty_commit_search_zwz/6
image

### CODE REVIEW

File: scripts/modelbench/inductor_single_run.sh [hunk 1]
```diff
@@ -51,11 +44,24 @@ if [[ "$DT" == "amp_fp16" ]]; then
 fi
 
 Flag_extra=""
-if [[ ${FREEZE} == "on" ]]; then
+cd /workspace/pytorch/benchmarks/dynamo
+if [[ $BACKEND == "aot_inductor" ]]; then
+    echo "Testing with aot_inductor."
```
Comment (chuanqi129) [inline scripts/modelbench/inductor_single_run.sh:49]: Those changes look a little bit complex, could we define a `${backend_extra}` to handle different backend?

Comment (WeizhuoZhang-intel) [reply scripts/modelbench/inductor_single_run.sh:49]: Submit a PR for this modification: https://github.com/chuanqi129/inductor-tools/pull/132 
1. `Backend_extra` is defined and replaced `Flag_extra`
2. Set `Freezing` by default for all backend

Comment (WeizhuoZhang-intel) [reply scripts/modelbench/inductor_single_run.sh]: Fixed by https://github.com/chuanqi129/inductor-tools/pull/132

**Review by chuanqi129 — APPROVED** [review]
May need to add freezing back for new requirements

**Pull request merged**